tests/cypress/support/commands.js

```diff
@@ -739,10 +749,11 @@ Cypress.Commands.add('createAndPublishApi', (apiName = null) => {
 })
 
 Cypress.Commands.add('logoutFromDevportal', (referer = '/devportal/apis') => {
-    cy.visit(`${Utils.getAppOrigin()}/devportal/apis?tenant=carbon.super`);
-    cy.wait(2000);
+    //cy.visit(`${Utils.getAppOrigin()}/devportal/apis?tenant=carbon.super`);
```

Comment:
Let's remove commented code

Reply:
this has already been fixed by a later commit